spec/requests/requests_spec.rb

```diff
@@ -3,85 +3,243 @@
+        subject.call
+        expect(response).to be_not_found
+      end
+    end
+
+    context 'authenticated and authorized' do
+      let(:user) { create(:user, organizations: [organization]) }
+
+      context 'request not part of organization' do
+        let(:request) { create(:request, organization: create(:organization)) }
+
+        it 'renders not found ' do
+          subject.call
+          expect(response).to be_not_found
+        end
+      end
+
+      context 'request is part of organization' do
+        it 'should be successful' do
+          subject.call
+          expect(response).to be_successful
         end
+      end
+    end
+  end
+
+  describe 'POST /{organization_id}/requests' do
+    subject { -> { post organization_requests_path(organization, as: user), params: params } }
+
+    let(:params) { { request: { title: 'Example Question', text: 'How do you do?', hints: ['confidential'] } } }
+    let(:user) { create(:user, organizations: [organization]) }
+
+    before do
+      allow(Request).to receive(:broadcast!).and_call_original # is stubbed for every other test
+    end
+
+    context 'unauthenticated' do
+      let(:user) { nil }
+
+      it 'redirects to the sign in path' do
+        subject.call
+        expect(response).to redirect_to(sign_in_path)
+      end
+    end
+
+    context 'unauthorized' do
+      let(:user) { create(:user, organizations: [create(:organization)]) }
+
+      it 'renders not found ' do
+        subject.call
+        expect(response).to be_not_found
+      end
+    end
+
+    context 'authenticated and authorized' do
+      it { should change { Request.count }.from(0).to(1) }
+
+      it 'redirects to requests#show' do
+        subject.call
+        request = organization.requests.first
+        expect(response).to redirect_to organization_request_path(organization, request)
+      end
+
+      it 'shows success notification' do
+        subject.call
+        expect(flash[:success]).to eq('Deine Frage wurde erfolgreich an 0 Mitglieder in der Community gesendet')
+      end
+
+      describe 'without hints param' do
+        let(:params) { { request: {  title: 'Example Question', text: 'How do you do?' } } }
+        it { should_not raise_error }
+      end
 
-        it 'shows success notification' do
+      describe 'without contributors' do
+        it { should_not raise_error }
+      end
+
+      describe 'with contributors' do
+        before do
+          create_list(:contributor, 2, organization: organization)
+          create(:contributor, organization: create(:organization))
           subject.call
-          expect(flash[:success]).not_to be_empty
+        end
+
+        context 'with image file(s)' do
+          let(:params) do
+            { request: { title: 'Message with files', text: 'Did you get this image?',
+                         files: [fixture_file_upload('profile_picture.jpg')] } }
+          end
+
+          describe 'an image file' do
```

Comment:
```suggestion
          describe 'one image file' do
```
Is there a significant difference btw. one and many image files that justifies its own block & specs

Reply:
I added these specs 2 years ago. Looking the logic, it doesn't appear to test anything different, so I would say it's not needed. Since we are doing such a big refactor of these specs anywhere, I'll remove on of them vs doing it in a follow-up PR.

Reply:
As far as your suggestion to improve the English. This is not an improvement as _an_ can only mean _one_